### ISSUE TITLE
SA1625 Should not report multiple params with inheritdoc tag as copy pasted

### ISSUE THREAD

**Issue by domn1995** [DotNetAnalyzers/StyleCopAnalyzers#3353]
Given a documented interface:
```cs
interface IFoo 
{
    /// <summary>Does something.</summary>
    /// <param name="param1">The first parameter.</param>
    /// <param name="param2">The second parameter.</param>
    void DoSomething(string param1, string param2);
}
```

When I implement that interface and write my own documentation:
```cs
class Foo : IFoo
{
    /// <summary>The Foo way of doing something.
    /// <param name="param1"><inheritdoc/></param>
    /// <param name="param2"><inheritdoc/></param>
    void DoSomething(string param1, string param2)
    {
        System.Console.WriteLine("Doing something...");
    }  
}
```

Then:
SA1625 should not be raised on the second parameter (or additional parameters) of `Foo`. I assume this issue occurs when extending a class as well but I haven't tested it.

**Comment by sharwell** [comment]
Duplicate of #3108

**Comment by sharwell** [comment]
📝 I recommend this form, which inherits everything except for `<summary>` and avoids the warnings:

```cs
class Foo : IFoo
{
    /// <summary>The Foo way of doing something.
    /// <inheritdoc/>
    public void DoSomething(string param1, string param2)
    {
        System.Console.WriteLine("Doing something...");
    }  
}
```

**Comment by domn1995** [comment]
Thank you! I didn't know you could inheridoc this way :)
